elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-indeterminate]) {
+    animation: spin 1s linear infinite;
+    -webkit-animation:spin 1s linear infinite;
+    -moz-animation:spin 1s linear infinite;
+
+    border-bottom: 4px solid rgba(0,0,0,0.25);
+    border-left: 4px solid rgba(0,0,0,0.25);
+    border-radius: 100%;
+    border-right: 4px solid rgba(0,0,0,0.25);
+    border-top: 4px solid rgba(0,0,0,0.75);
+
+    display: inline-block;
+    height: 2rem;
+    margin: 0 auto;
+    position: relative;
+    width: 2rem;
+    vertical-align: middle;
+    visibility: visible;
+}
+
+@-ms-keyframes spin {
```

Comment:
Since a lot of these values are repeated I wonder if we could set some local variables for these or possibly include them in a mixin with an @for to loop through the different vendor prefixes?

Reply:
I think once we have auto-prefixer that will be a moot point. At the moment it's four rules, but I think in the future it will just be one.

Reply:
Autoprefixer is already added so you can remove this code :)